infra/docker-images/db-ops/dump_restore.sh

```diff
@@ -3,118 +3,149 @@
 # pcapi app expects a DATABASE_URL environment variable
 export DATABASE_URL=${PUBLIC_DATABASE_URL}
 export POSTGRES_CONNEXION_STRING_DEST=${PUBLIC_DATABASE_URL}
+export DUMP_BUCKET_PATH="$(date +'%y-%m-%d')_${DUMP_BUCKET_PATH}"
```

Comment:
Ne reutilise pas la meme variable:
```suggestion
export TIMESTAMPED_DUMP_BUCKET_PATH="$(date +'%y-%m-%d')_${DUMP_BUCKET_PATH}"
```

Reply:
Et si on utilise la variable DUMP_BUCKET_PATH c'est pas clair. Si j'y mets un truc du genre `dir/dump.gs` ca va timestamper le repertoire, ce qui est pas forcement ce qu'on veut

Reply:
Bien vu mais la suggestion ne corrige pas le problème
Insérer une var dir/ et une var file_name ?

Reply:
Yiss sire

Reply:
fait

Reply:
Toujours chaud pour que tu affectes pas une nouvelle valeur à la même variable